packages/client/src/components/notebook/NotebookCellConsole.tsx

```diff
@@ -15,14 +16,35 @@ interface ConsoleProps {
 
 export const NotebookCellConsole = (props: ConsoleProps) => {
     const { messages } = props;
+    const isOutputJSON = (str: string) => {
```

Comment:
Is this a utility function that can be used elsewhere in the application? If so, move it into a util folder -- otherwise disregard.

Reply:
Moved to utility folder.